docs/source/10-minutes-to-dask.rst

```diff
@@ -406,6 +408,7 @@ triggering computation, we can inspect the task graph to figure out what's going
          >>> c.visualize()
 
       .. image:: images/10_minutes_bag_graph.png
+         :alt: A two-dimensional grid of blocks where the borders of each block have thin strips representing data shared from its neighbors. There are also small corner bits representing data shared from diagonal neighbors.
```

Comment:
I don't think this alt text belongs to this image? Is it supposed to go with one of the other images from the accessibility sprint instead?

This image of a dask bag task graph `images/10_minutes_bag_graph.png` is embarrassingly parallel, it doesn't share anything with neighbours as far as I can tell.

Reply:
```suggestion
         :alt: Dask task graph for a Dask bag computation. The task graph shows a "lambda" operation, and then a "zip" operation is applied to the partitions of the Dask bag. There is no communication needed between the bag partitions, this is an embarrassingly parallel computation. This computation can be written as "db.zip(b, b.map(lambda x: x * 10))".
```